src/routers/productExports.ts

```diff
@@ -7,14 +7,110 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /productexports
+ * @description Returns all product exports and all their associated attributes.
+ *              Guaranteed fields are:
+ *                - productTransactionUuid
+ *                - productType
+ *                - datetime (UTC)
+ *                - toPersonUuid
+ *                - fromPersonUuid
+ *                - amountOfProduct
+ *                - productUnits
+ *                - costPerUnit
+ *                - currency
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /products/
+ * @apiName GetProductExports
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all product exports
+ * @apiSuccessExample Success-Response:
+  [
+    {
+      "productTransactionUuid": "9f11efb5-d5b4-4853-aa88-ee10c2940c9f",
+      "datetime": "2018-01-23 04:05:06",
+      "toPersonUuid": "34e0574e-beea-498d-8bec-7f26d84ba761",
+      "fromPersonUuid": "95bb2019-d135-49a8-83a7-4bf8f54684a0",
+      "productType": "milk",
+      "amountOfProduct": 10.23,
+      "productUnits": "litres",
+      "costPerUnit": 22.56,
+      "currency": "UGX",
+      "density": "5"
```

Comment:
maybe provide density units?

Reply:
do we need density units?

Reply:
No, this is an extra attribute, so anything goes.